lib/routes/css-tricks/guides.ts

```diff
@@ -0,0 +1,39 @@
+import { Data, Route, ViewType } from '@/types';
+import { extractMiniCards, processCards, rootUrl } from './utils';
+export const route: Route = {
+    path: '/guides',
+    view: ViewType.Articles,
+    categories: ['programming'],
+    example: '/guides',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['css-tricks.com'],
+            target: '/guides',
+        },
+    ],
+    name: 'CSS Guides',
+    maintainers: ['Rjnishant530'],
+    handler,
+};
+
+async function handler() {
+    const guideCards = await extractMiniCards('body > div.page-wrap > section.post-sliders > div:nth-child(3) article.mini-card.module.module-article');
+    const items = await processCards(guideCards, true);
```

Comment:
Please use [WordPress REST API](https://developer.wordpress.org/rest-api/) instead.
The API endpoints are available at `/wp-json/wp/v2`.

Once you've obtained all the post ids, you can use `/wp/v2/posts` with `includes` search parameter to obtain all the data in 1 HTTP request.